### PR TITLE
update some imaginary dependencies to improve performance

### PR DESCRIPTION
**Pull request by szaimen**
Close part of  https://github.com/nextcloud/server/issues/35643#issuecomment-1455137443

<details>
<summary>Speed for reference image until now:</summary>

```
172.18.0.9 - - [07/Mar/2023 08:40:09] "POST /pipeline?operations=%5B%7B%22operation%22%3A%22autorotate%22%7D%2C%7B%22operation%22%3A%22fit%22%2C%22params%22%3A%7B%22width%22%3A2048%2C%22height%22%3A2048%2C%22stripmeta%22%3A%22true%22%2C%22type%22%3A%22jpeg%22%2C%22norotation%22%3A%22true%22%2C%22quality%22%3A%2260%22%7D%7D%5D HTTP/1.1" 200 24881 0.7438
```

</details>

### CODE REVIEW

**Comment by Zoey2936** [comment]
why sid? sid is called experimental currently for reasons

**Comment by szaimen** [comment]
> why sid? sid is called experimental currently for reasons

It was recommended in https://github.com/nextcloud/server/issues/35643#issuecomment-1455137443. Do you think we should rather go with backports instead or not do this at all?

**Comment by Zoey2936** [comment]
https://packages.debian.org/bullseye/libheif1 v1.11
https://packages.debian.org/bookworm/libheif1 v.14
https://packages.debian.org/sid/libheif1 v1.15

**Comment by Zoey2936** [comment]
would bookworm maybe be the better way? and would it not be better to add the packages list only for installing the packages and then removing them?

**Comment by szaimen** [comment]
> would bookworm maybe be the better way? 

Sounds good!

> and would it not be better to add the packages list only for installing the packages and then removing them?

wdym?

**Comment by szaimen** [comment]
> deleting the /etc/apt/sources.list.d/bookworm.list file after installing and adding it first after installing the netcat package

good point, done👍

File: Containers/imaginary/Dockerfile [hunk 1]
```diff
@@ -8,6 +8,15 @@ RUN set -ex; \
     apt-get install -y --no-install-recommends \
         netcat \
     ; \
+    echo "deb http://deb.debian.org/debian bookworm main" > /etc/apt/sources.list.d/bookworm.list \
+    apt-get update; \
+    apt-get install -t bookworm -y --no-install-recommends \
+        libheif1 \
+        libde265-0 \
+        libx265-199 \
+        libvips \
```
[the remark this answers is not in the document]
Comment (szaimen) [reply Containers/imaginary/Dockerfile:17]: > Uploading 959B7D29-15C3-4698-B601-BBF151746B21.png…

I fear that didnt work...

[the remark this answers is not in the document]
Comment (Zoey2936) [reply Containers/imaginary/Dockerfile:17]: 2BAB1366-5393-44AA-A138-B4645764A661

[the remark this answers is not in the document]
Comment (szaimen) [reply Containers/imaginary/Dockerfile:17]: I see. So it is a symlink to libvips42

**Comment by szaimen** [comment]
Thanks Zoey! :)

**Comment by szaimen** [comment]
@Zoey2936 this seems to fail: https://github.com/nextcloud-releases/all-in-one/actions/runs/4352742007/jobs/7605899041

Do you have any idea what is wrong and how to fix this?

**Comment by szaimen** [comment]
Found it: https://github.com/nextcloud/all-in-one/commit/671a73a6c4b0f580a61c738efb3ca3d0f9727adb

**Pull request merged**